### PR TITLE
Initialize repo

### PR DESCRIPTION
**Pull request by kaneeldias**
## Purpose
$subject

Related issue: https://github.com/ballerina-platform/ballerina-standard-library/issues/4488

### CODE REVIEW

File: ballerina/Ballerina.toml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+version = "1.0.0"
+authors = ["Ballerina"]
+keywords = ["persist", "sql", "mysql", "experimental"]
+repository = "https://github.com/ballerina-platform/module-ballerinas-persist.sql"
+icon = "icon.png"
+license = ["Apache-2.0"]
+distribution = "2201.6.0"
+
+[[platform.java11.dependency]]
+groupId = "io.ballerinax.persist.sql"
```
Comment (daneshk) [inline ballerina/Ballerina.toml:13]:
```suggestion
groupId = "io.ballerina.stdlib"
```

File: build-config/resources/Ballerina.toml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+version = "@toml.version@"
+authors = ["Ballerina"]
+keywords = ["persist", "sql", "mysql", "experimental"]
+repository = "https://github.com/ballerina-platform/module-ballerinas-persist.sql"
+icon = "icon.png"
+license = ["Apache-2.0"]
+distribution = "2201.6.0"
+
+[[platform.java11.dependency]]
+groupId = "io.ballerinax.persist.sql"
```
Comment (daneshk) [inline build-config/resources/Ballerina.toml:13]:
```suggestion
groupId = "io.ballerina.stdlib"
```

File: build-config/resources/Ballerina.toml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+artifactId = "persist.sql-native"
+version = "@toml.version@"
+path = "../native/build/libs/persist.sql-native-@project.version@.jar"
+
+[[platform.java11.dependency]]
+groupId = "io.ballerina.stdlib.persist"
```
Comment (daneshk) [inline build-config/resources/Ballerina.toml:19]:
```suggestion
groupId = "io.ballerina.stdlib"
```

Comment (kaneeldias) [reply build-config/resources/Ballerina.toml]: Fixed in [c887e05](https://github.com/ballerina-platform/module-ballerinax-persist.sql/pull/1/commits/c887e05a70d6e20c0fcca886464b990a2dd6e1f7)

File: build-config/resources/Ballerina.toml (hunk 2 above)
Comment (kaneeldias) [reply build-config/resources/Ballerina.toml]: Fixed in [c887e05](https://github.com/ballerina-platform/module-ballerinax-persist.sql/pull/1/commits/c887e05a70d6e20c0fcca886464b990a2dd6e1f7)

File: ballerina/Ballerina.toml (hunk 1 above)
Comment (kaneeldias) [reply ballerina/Ballerina.toml]: Fixed in [c887e05](https://github.com/ballerina-platform/module-ballerinax-persist.sql/pull/1/commits/c887e05a70d6e20c0fcca886464b990a2dd6e1f7)

**Review by daneshk — APPROVED** [review]

**Review by daneshk — APPROVED** [review]

**Pull request merged**